sopel/modules/url.py

```diff
@@ -353,16 +358,30 @@ def title_auto(bot: SopelWrapper, trigger: Trigger):
         bot.memory["last_seen_url"][trigger.sender] = url
 
 
+class URLInfo(NamedTuple):
+    """Helper class for information about a URL handled by this plugin."""
+
+    url: str
+
+    title: Optional[str]
+    """The title associated with ``url``, if appropriate."""
+
+    hostname: Optional[str]
+    """The hostname associated with ``url``, if appropriate."""
+
+    tinyurl: Optional[str]
+    """A shortened form of ``url``, if appropriate."""
```

Comment:
This is just following the rest of the code's name for the returned value, and _at present_ it is still an actual `tinyurl` from the TinyURL service. I just wonder how long that's going to last, since the API endpoint this plugin uses appears not to be advertised anywhere on tinyurl.com (that I could find).

But if the TinyURL backend stops working, we can rename this and every other related variable when that time comes. No need to bother with it now.